### ISSUE TITLE
Add 'curator note' property to selected classes

### ISSUE THREAD

**Issue by cmungall** [geneontology/go-ontology#13384]
On the managers call http://wiki.geneontology.org/index.php/Manager_Call_2017-04-19#Documentation

@pgaudet suggested improving curator documentation centered around particular terms in the ontology. I propose that we embed this directly in the ontology.

We are no longer restricted to the grab-bag 'comment' field for additional notes on a class, so I propose we create a new property or reuse something like [IAO:0000232](http://purl.obolibrary.org/obo/IAO_0000232) `curator note`

We could add axiom annotations to each such owl annotation assertion indicating if and how this should be propagated down to descendant terms.

For some of these, it could go in the design pattern too.

By default a non-comment property will not be visible in most browsers, but these could be configured to add these.

Would this work for P2Go @tonysawfordebi?

**Comment by ukemi** [comment]
Discussed on the ontology editors call. We think this is a good idea. How do you find the relevant notes for child terms when the note is on a parent?
 For example, don't annotate a secreted protein to Golgi.

**Comment by cmungall** [comment]
We will have a SPARQL construct that will be run periodically to propagate these down in the editors ontology. @dougli1sqrd can implement.

**Comment by cmungall** [comment]
@geneontology/ontology - can you provide @dougli1sqrd an example of a curator note and its associated GO ID so we can get started on this.

**Comment by ukemi** [comment]
[Term]
id: GO:0000084
name: mitotic S phase
namespace: biological_process
def: "The cell cycle phase, following G1, during which DNA synthesis takes place as part of a mitotic cell cycle." [GOC:mtg_cell_cycle]
comment: Note that this term should not be used for direct annotation. If you are trying to make an annotation to x phase, it is likely that the correct annotation is 'regulation of x/y phase transition' or to a process which occurs during the reported phase (i.e mitotic DNA replication for mitotic S-phase). To capture the phase when a specific location or process is observed, the phase term can be used in an annotation extension (PMID:24885854) applied to a cellular component term (with the relation exists_during) or a biological process term (with the relation happens_during).
subset: gocheck_do_not_annotate

**Comment by pgaudet** [comment]
@ukemi Should we make this 'high priority'?

**Comment by ukemi** [comment]
We need examples first, don't you think?

**Comment by pgaudet** [comment]
I thought your comment above was an example.

**Comment by ukemi** [comment]
OK. But I thought there was the additional step of adding a different kind of comment for obsoletions, ontology editor comments etc. We need to specify the requirements a little more.

**Comment by pgaudet** [comment]
You mean we want more than one new type of comment? I thought this all
going to be lumped under comment for curator.

Le jeu. 31 mai 2018 à 3:02 PM, David Hill <notifications@github.com> a
écrit :

**Comment by ukemi** [comment]
I didn't think so. Perhaps this is a topic for an ontology call.

**Comment by cmungall** [comment]
@ukemi you provided `mitotic S phase` as an example

Did you intend to provide GO:0044848 ! biological phase as the example, with the intent that the same curator note is propagated down to all subclasses?

**Comment by ukemi** [comment]
Yes. The point was the propagation of curator notes to children following specific properties.

**Comment by balhoff** [comment]
We recently added [editor note](http://purl.obolibrary.org/obo/IAO_0000116). I don't see how this differs from [curator note](http://purl.obolibrary.org/obo/IAO_0000232). But we aren't doing any of this propagation. Is this still desired?
